### DISCUSSION TITLE
Possible to edit all layers/frames inside a group at once?

### DISCUSSION THREAD

**Discussion by Guccisocks** [keiwando/toonsquid#94] [General]
Hi, absolutely love using toonsquid but wanted to know if its possible to select multiple pixel frames/layers and move then around?
So far after selecting multiple frames I'm only able to move one frame and not the whole selection. 

Any advice appreciated

**Comment by keiwando** [comment]
Hi! This is not possible yet but it's already on my list of features to consider for future updates. Thanks for the suggestion!
